src/components/pages/blog/posts.tsx

```diff
@@ -19,7 +19,7 @@ const BlogTeaserGrid = styled.div`
   // we fetch teaser of size 600px
   // this means we can search for a column size of 300px - 12px (half gap) = 288px
   // to auto fit our teasers
-  grid-template-columns: repeat(auto-fill, minmax(288px, 1fr));
+  grid-template-columns: repeat(auto-fill, minmax(278px, 1fr));
```

Comment:
Hey @georgiee,
I just wanted to ask about the comments here.
Here is the grid to store blog posts, and to fit them automatically, it uses a fixed width (288px). I did the same thing, just used a smaller value (278px) from the Figma layout to accommodate 4 articles in a one row.
In your comment you determined this value by calculation, but I did not understand the meaning of the calculation.
So I am not sure if my solution works. 
Is it ok if I just use the value from Figma?

Reply:
Oh I can't even remember I wrote this 😅 I think whatever works is good enough here. And if you can now fit 4 columns in a row then it's a good choice. When you have some spare time you can also try to understand the underlying auto fill algorithm and maybe answer it yourself by combining that knowledge with the figma design. Thanks for asking 👍